optax/contrib/_ademamix.py

```diff
@@ -1,39 +1,38 @@
-"""AdeMAMix.
+"""AdEMAMix.
 
 Implementation of
 "THE ADEMAMIX OPTIMIZER: BETTER, FASTER, OLDER"
 (https://arxiv.org/pdf/2409.03137) by Matteo Pagliardini, 
 Pierre Ablin and David Grangier.
 """
-import optax.tree_utils as otu
-from typing import NamedTuple, Optional
+
 import chex
 import jax.numpy as jnp
 import jax.tree_util as jtu
 from optax._src import base
 from optax._src import combine
 from optax._src import numerics
 from optax._src import transform
-
+import optax.tree_utils as otu
+from typing import NamedTuple, Tuple
```

Comment:
I think the convention in this library is that imports from python generally go at the top, so
```
from typing import...

import chex
...
```

Reply:
Ah!  Good point.  (I was going to make a meta point about this repo not having any `pre-commit` hooks but...).